src/Form/Themes.php

```diff
@@ -26,11 +26,38 @@ class Themes {
 
 
 	/**
-	 * Get Registered themes
+	 * Themes constructor.
+	 */
+	public function __construct() {
+	}
```

Comment:
Now you can remove this since you're no longer using it.

Reply:
https://github.com/impress-org/givewp/compare/b6184d567284...40544c3ae37d